Figgle.Generator/EmbedFontSourceGenerator.cs

```diff
@@ -0,0 +1,329 @@
+﻿// Copyright Drew Noakes. Licensed under the Apache-2.0 license. See the LICENSE file for more details.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Figgle.Fonts;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Figgle.Generator;
+
+[Generator(LanguageNames.CSharp)]
+internal sealed class EmbedFontSourceGenerator : IIncrementalGenerator
```

Comment:
Now that we have `EmbedFontSourceGenerator` it doesn't seem that the other `FiggleSourceGenerator`'s name is descriptive/distinctive enough. What do you think about renaming it to `RenderTextSourceGenerator` or something like that?

Reply:
Yep makes sense!  I've renamed the class and associated test classes.